modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/KubernetesRuntimeProvisioningService.java

```diff
@@ -342,20 +342,207 @@ public DeploymentLogs getRuntimeLogs(DeploymentConfig deploymentConfig, Query qu
         return deploymentLogs;
     }
 
+    /**
+     * Set runtime properties to kubernetes environment
+     *
+     * @param runtimeProperties runtime properties
+     * @param deploymentConfig  includes deployment related details
+     * @throws RuntimeProvisioningException
+     */
     @Override
-    public void setRuntimeProperties(List<RuntimeProperty> runtimeProperties)
+    public void setRuntimeProperties(List<RuntimeProperty> runtimeProperties, DeploymentConfig deploymentConfig)
             throws RuntimeProvisioningException {
 
+        List secrets = new ArrayList();
```

Comment:
This is fixed in following commit.
https://github.com/wso2/product-af/pull/800/commits/7d70ba5f5bbacfe7789db67072db49c2ff893b78